keras_nlp/models/bert/bert_tasks.py

```diff
@@ -101,11 +101,13 @@ def __init__(
         self,
         backbone,
         num_classes=2,
+        dropout=0.1,
```

Comment:
Can we just not expose this for now and read the property of the backbone?

```
pooled = backbone(inputs)["pooled_output"]
pooled = keras.layers.Dropout(backbone.dropout)(pooled)
```

Reply:
I think we should give users freedom to adjust the dropout value?

Reply:
If we want it configurable, you could still have it default to the backbone value (even for our presets). Something like...

```python
def __init__(..., dropout=None):
    ...
    pooled = backbone(inputs)["pooled_output"]
    # Default to the same dropout value as our backbone
    dropout = dropout or backbone.dropout
    pooled = keras.layers.Dropout(dropout)(pooled)
```

This was you would not need to double up the parameter in the preset config. But if someone for some reason wanted a different value, you would have the flexibility to do so.

Reply:
Would anyone confuse `dropout=None` with `dropout=0.0`?

Reply:
Perhaps!

Also just remembered we have this already on all our other heads. So maybe scratch my whole suggestion at least just for consistency.

Reply:
I don't think defaulting the dropout value to backbone is necessary, actually I think it could create confusion on users because this default is hidden logic. 

See TFM's implementation: https://www.tensorflow.org/api_docs/python/tfm/nlp/models/BertClassifier

Having an explicit default is clear.

Reply:
Fine with me! But then we should remove the line Jonathan is suggesting above.